src/app/app.py

```diff
@@ -73,19 +100,30 @@ async def get_schema(survey_id: str, version: str) -> dict:
 
     schema = storage.get_schema(schema_metadata.schema_location)
 
+    if schema is None:
+        logger.error("Schema not found")
+        return exception_throw.throw_404_no_schema_exception()
+
     logger.info("Schema successfully retrieved.")
     logger.debug(f"Schema: {schema}")
 
     return schema
 
 
 @app.get("/v1/schema_metadata", response_model=list[ReturnedSchemaMetadata])
-async def get_schemas_metadata(survey_id: str) -> list[ReturnedSchemaMetadata]:
+async def get_schemas_metadata(survey_id: str = "") -> list[ReturnedSchemaMetadata]:
```

Comment:
How come we're setting a default value here? What happens if we remove this then on line 116 check 
`if survey_id is None`
instead?

Reply:
Changed from empty string to None